### PR TITLE
Updated to latest Bootstrap v3

### PR DESCRIPTION
**Pull request by jlowe64**
Updated to Bootstrap latest v3
Add Bootstrap fonts
Adjusted template grids

### CODE REVIEW

**Comment by jlowe64** [comment]
Hey, could you double check this on your end? I have checked all the classes I could see and also the conflicts you mentioned in your email. I didn't find anything of similar name in Bootstrap.

File: app/views/layout.jade [hunk 1]
```diff
@@ -51,17 +51,21 @@ html
...
-              | Powered by <a href="http://github.com/benjie/members-area">Members Area</a>.
-    
+            .row
+              .footer-col
+                .col-md-6
+                  a(href='#') Back to top
+              .footer-col
+                .col-md-6
+                  p
+                    | Powered by <a href="http://github.com/benjie/members-area">Members Area</a>.
```
Comment (benjie) [inline app/views/layout.jade]: While you're at it, fancy fixing this link? `members-area/members-area`

File: app/views/registration/register.jade [hunk 2]
```diff
@@ -54,8 +54,8 @@ block body
             label(class="text-"+controlGroupClass)
               input(type="checkbox", name="terms", tabindex=tabIndex++)
               |  I confirm that the information I have entered above is not
-              |  fraudulent, and that I agree to be bound by the 
+              |  fraudulent, and that I agree to be bound by the
```
Comment (benjie) [inline app/views/registration/register.jade]: Be very very careful of trimming trailing whitespace in Jade - in this case, it's omission will result in the word "the" merging with the link text from the next line.

File: public/js/npm.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+require('../../js/button.js')
+require('../../js/carousel.js')
+require('../../js/collapse.js')
+require('../../js/dropdown.js')
+require('../../js/modal.js')
+require('../../js/tooltip.js')
+require('../../js/popover.js')
+require('../../js/scrollspy.js')
+require('../../js/tab.js')
+require('../../js/affix.js')
```
Comment (benjie) [inline public/js/npm.js]: I don't think we need this file

**Comment by benjie** [comment]
Excellent work; this looks harmless enough! Just two things to do:

- [ ] fix the removed whitespace mentioned above
- [ ] remove the unnecessary npm.js file

I'll take it for a spin when I get a chance and all going well (which I suspect it will) I'll merge it - thanks very much! :grinning:

File: app/views/registration/register.jade (hunk 2 above)
Comment (jlowe64) [inline app/views/registration/register.jade]: Ahh I'm assuming Atom did that on purpose. Must be a setting. I don't remember touching that area. Will fix it, though.

**Comment by benjie** [comment]
Perfect :ok_hand:

**Comment by benjie** [comment]
Whoops; thought I'd merged this already - sorry!

**Pull request merged**